CTW/Outback/map.json

```diff
@@ -78,8 +78,8 @@
 		}
 	],
 	"itemremove": [
-		"iron sword", "bow", "diamond pickaxe", "iron axe", "leather helmet", "glass", "oak planks", "cooked beef", 
-		"arrow", "leather helmet", "chainmail chestplate", "leather leggings", "iron boots", "beacon", "string"
+		"iron sword", "bow", "diamond pickaxe", "iron axe", "glass", "oak planks", "cooked beef", 
+		"arrow", "leather helmet", "chainmail chestplate", "leather leggings", "iron boots", "beacon", "string", "golden apple"
```

Comment:
You could make it nicer to the eyes here by just putting "golden apple" on the first line so that each line is even.